projects/kubernetes/e2e/Makefile

```diff
@@ -0,0 +1,29 @@
+ORG?=docker
+IMAGE=kubernetes-for-desktop-e2e
```

Comment:
Is it specific to `desktop` though? Couldn't it just as well run on a LinuxKit VM running on a Linux host?

Reply:
Practically, it's not, apart from the hardcoded IP.